manager/backingimage.go

```diff
@@ -88,6 +88,13 @@ func (m *VolumeManager) CreateBackingImage(name, checksum, sourceType string, pa
 }
 
 func (m *VolumeManager) DeleteBackingImage(name string) error {
+	replicas, err := m.ds.ListReplicasByBackingImage(name)
```

Comment:
Why we move the validation back here from the admission webhook?

Reply:
The checks in DELETE are not general. Uninstallation does not need the checks.
https://github.com/longhorn/longhorn-manager/blob/master/controller/uninstall_controller.go#L361

Reply:
As discussed, `backing image validation` looks general, but the validation of node & engine images moved out from the admission look good. 

Also, suggest adding a comment to the README to mention when adding validation to deletion, we should be aware of this possible side effect.